lib/rules/space-infix-ops.js

```diff
@@ -21,25 +38,33 @@ module.exports = {
         schema: [
             {
                 type: "object",
-                properties: {
-                    int32Hint: {
-                        type: "boolean"
-                    }
-                },
+                properties: schemaProperties,
                 additionalProperties: false
             }
         ]
     },
 
     create(context) {
-        const int32Hint = context.options[0] ? context.options[0].int32Hint === true : false;
-
-        const OPERATORS = [
-            "*", "/", "%", "+", "-", "<<", ">>", ">>>", "<", "<=", ">", ">=", "in",
-            "instanceof", "==", "!=", "===", "!==", "&", "^", "|", "&&", "||", "=",
-            "+=", "-=", "*=", "/=", "%=", "<<=", ">>=", ">>>=", "&=", "^=", "|=",
-            "?", ":", ",", "**"
-        ];
+        const option = context.options[0] || {};
+        const int32Hint = Boolean(option.int32Hint);
```

Comment:
Not 100% sure if this is what ljharb was getting at, but `Boolean(x)` is vulnerable to global reassignment (e.g., if someone did `Boolean = function someOtherBooleanConstructor() {}` elsewhere in the code). `!!x` is impervious to that sort of thing because operators can't be redefined in JavaScript.